web/src/main/java/se/inera/intyg/rehabstod/service/sjukfall/PopulateFiltersService.java

```diff
@@ -19,8 +19,12 @@
 
```

Comment:
We should switch the package of this to a common one. Maybe filter instead of sjukfall?

Reply:
Good suggestion, will fix this!